cmd/send-email/main.go

```diff
@@ -25,7 +28,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
-    err = email.Send(args.To, args.Subject, args.Body, false);
+    // If database cannot get course and using role based addressing
+    // Then, resolve users will return an error
+    // So, we do not error check on the get course method
+    course, err := db.GetCourse(args.Course);
+    emailTo, err := db.ResolveUsers(course, args.To);
+    if (err != nil) {
+        log.Fatal("Failed to resolve users:'%s', '%w'.", course.GetName(), err);
+    }
```

Comment:
You will want to separate independent if statements with vertical whitespace.

This falls under the category of "transparent coding", which I will mention sometimes.
It is about making your code as clear and readable as possible.
By having a whitespace here, it makes it easier for a reader to see that these are two different cases.
Otherwise, it is really easy for a reader to accidentally see an `if` as an `else if` and think that these cases are linked.